configs/raspberrypi_64_config

```diff
@@ -0,0 +1,107 @@
+# Binaries generated with the following script:
+#    https://github.com/mendersoftware/mender-convert-integration-scripts/blob/master/build-uboot-rpi64.sh
+
+ASSETS_PATH="rpi_arm64"
```

Comment:
Remember to remove these lines

Reply:
@lluiscampos actually I'd vote for keeping the asset pulling logic, and just remove this `ASSETS_PATH` assignment to provide an easy way for users to inject their custom `u-boot` if required.